server/data/manageUsersApiClient.ts

```diff
@@ -3,31 +3,27 @@ import RestClient from './hmppsRestClient'
 import { User } from '../@types/CvlUserDetails'
 import { TokenStore } from './tokenStore'
 
-export type AuthUserDetails = {
+export type PrisonUserDetails = {
```

Comment:
renamed to PrisonUserDetails as UserDetails exists on the Community API and wanted to avoid confusion between types